### PR TITLE
Boot speedup

### PR DESCRIPTION
**Pull request by rotorman**
Removes various delays from SDRAM initialization (by comparing with how STM32F4 HAL initializes FMC, I see no delays being used).

Tested with RM TX16S, boots ca. 3 times faster than before.

NOT tested with NV14 yet!

### CODE REVIEW

**Comment by pfeerick** [comment]
TX16S does indeed boot significantly faster... so much so that the dreaded back screen and arrow is now more noticeable if modelbmp has a moderate sized image to load from SD 😆 NV14 appears to be fine. It does seem faster also, but it still has that massive 4 second delay before showing any signs of life...

File: radio/src/targets/horus/sdram_driver.c [hunk 1]
```diff
@@ -194,59 +184,45 @@ void SDRAM_InitSequence(void)
...
-  uint32_t timeout = SDRAM_TIMEOUT; 
   
-  /* Step 3 --------------------------------------------------------------------*/
   /* Configure a clock configuration enable command */
   FMC_SDRAMCommandStructure.FMC_CommandMode = FMC_Command_Mode_CLK_Enabled;
   FMC_SDRAMCommandStructure.FMC_CommandTarget = FMC_Command_Target_bank1;
   FMC_SDRAMCommandStructure.FMC_AutoRefreshNumber = 1;
   FMC_SDRAMCommandStructure.FMC_ModeRegisterDefinition = 0;
   /* Wait until the SDRAM controller is ready */ 
-  while((FMC_GetFlagStatus(FMC_Bank1_SDRAM, FMC_FLAG_Busy) != RESET) && (timeout > 0))
```
Comment (raphaelcoeffic) [inline radio/src/targets/horus/sdram_driver.c:206]: Not sure why it was there, I haven’t seen that anywhere else, but the timeout should not slow down anything. In the contrary, it should prevent waiting for ever.

**Comment by raphaelcoeffic** [comment]
> TX16S does indeed boot significantly faster... so much so that the dreaded back screen and arrow is now more noticeable if modelbmp has a moderate sized image to load from SD 😆 NV14 appears to be fine. It does seem faster also, but it still has that massive 4 second delay before showing any signs of life...

I think I will to trace that boot process in the NV14, as it seems really strange indeed. It takes forever.

**Comment by raphaelcoeffic** [comment]
Here is your power on delay of additional 3s:
0394847D-1313-4643-BB3F-4A38BB15260D

**Comment by rotorman** [comment]
I changed the value to calculated ca. 0.75 s on NV14 - could you please test if that makes an improvement on NV14. Thx!
(I guess cannot set POWER_ON_DELAY on NV14 to 0 ?)

**Comment by pfeerick** [comment]
I'll try it out shortly. Shortening would at least be a massive step, and if this PR doesn't make it into 2.5.0 (which I would respectfully suggest it shouldn't so we can move onto 2.6) and but this change has no ill effects, I'd like to cherry pick at least that commit for the NV14 for 2.5.0.

**Comment by pfeerick** [comment]
So, yes, it is speeding up NV14

This is some random pre PR commit...
https://youtu.be/rx5nQy9nMQg

This was the earlier commit
https://youtu.be/Go5dV6pt3hU

This was after the power-on delay change (yes, I finally unlocked the screen rotation so you don't have to look sideways 😆 )
https://youtu.be/B7qrvCSwsFE

I've not done anything more yet other than power on test. I think I'll see what 1 does later, as the NV14 does have dual power button protection, so there's basically no reason for the delay 😆.

**Comment by pfeerick** [comment]
Aw, shucks, you shouldn't have... this was waiting for me when I have finished doing some other stuff :)

That does seem quicker... and seems on par with the tx16s power on delay now. You'll just have to believe me for now (as I don't have a third arm to hold the camera :-P), but I actually just tried both the NV14 and TX16 at the same time, and there was an erie echo from the "welcome to edgetx" startup sound  ... is like100ms from being in perfect harmony 😆

Well, this time I even had a bound telemetry flysky receiver on, so it shows the internal rf is up and working also ;) 

https://youtu.be/FV3JKs_ySMs

**Comment by pfeerick** [comment]
My cat had something to say on that matter also 😆 
https://youtu.be/LOr2roF9Qeg

**Comment by CapnBry** [comment]
Oh this is amazing! I just flashed my TX16S + Bootloader with this PR applied and it is easily twice as fast as it was, and I don't have to hold the power button forever to boot.

**Comment by pfeerick** [comment]
Since this is hardware side, I defer to @raphaelcoeffic to sign off on this one, also given there was a comment still outstanding. Plus it now need de-conflicting with changes in NV14 code before merge - possibly complete removal of those changes since @raphaelcoeffic did a bit of work in that area recently, which reduced wait times for NV14. 

As far as I'm concerned, it worked perfectly though.

**Comment by raphaelcoeffic** [comment]
@rotorman I think we should tackle this in more invasive way, for example by adapting to the new HAL/LL drivers and make sure we respect the ST's precaution (no clue yet how that looks like).

**Pull request closed without merging**